docs/Makefile

```diff
@@ -31,7 +31,7 @@ setupenv:
 
 .PHONY: setup
 setup:
-	$(POETRY) install
+	$(POETRY) install --no-root
```

Comment:
Thanks @tchaikov! If possible, I'd prefer to add `package-mode = false` to the `pyproject.toml` file so that all Poetry-related configurations remain in the same file. Would you agree?

Reply:
i agree. i blindly followed the suggestion. and i was using poetry 1.7.1. so didn't look into other alternatives.